content/blog/announcing-crossguard-preview/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Announcing Crossguard Preview"
+date: 2019-12-02
+draft: true
+meta_desc: "Today we are announcing Pulumi CrossGuard, a Policy as Code solution, is available to all Team Pro and Enterprise customers to preview."
+meta_image: crossguard.svg
+authors: ["erin-krengel"]
+tags: ["crossguard", "policy-as-code", "New-Features", "Pulumi-News", "Features"]
+---
+
+Over the past few months, we have been hard at work on Pulumi CrossGuard, a Policy as Code solution. Using the `@pulumi/policy` library, you can express flexible business and security rules using code. CrossGuard enables organization administrators to enforce these policies across their organization or just on specific stacks. CrossGuard allows you to verify or enforce custom policies on changes before they are applied to your resources.
+
+<!--more-->
+
+You can run policies against any Pulumi stack, independent of the stack's language. This means you can have policies written in TypeScript and run them against a Python-based stack. Policies are grouped in *Policy Packs* and run against a stack during both `pulumi preview` and `pulumi up`. Each resource in a stack is run through the Policy Pack prior to creation, meaning you can block the creation of out-of-compliance resources.
+
+## Why
```

Comment:
Might be nice here to include a handful of examples across different dimensions: security, compliance, cost, team practices (e.g. tagging), etc.

Reply:
Added a paragraph! let me know you what you think